src/app/docs/text-highlight/text-highlight-docs.component.html

```diff
@@ -7,17 +7,54 @@
     The text highlight directive allows you to highlight text within DOM elements. When you set the skyHighlight attribute to the text you want to highlight, it highlights all matching text within the element. The text highlight directive can only be placed on elements that contain content, so you cannot place the directive on ng-template, ng-content, or ng-container tags.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-demo>
-  </sky-docs-demo>
+  <sky-docs-demo>
+    <div
+      class="sky-form-group"
+      style="width:200px;"
+    >
+      <label
+        [for]="searchTermInput.id"
+      >
+        Enter text to highlight:
```

Comment:
```suggestion
        Text to highlight
```
This change is more in line with our guidance at https://developer.blackbaud.com/skyux/design/guidelines/form-design#field-usability.